libs/launcher/multi_camera_launcher_base.cpp

```
@@ -119,8 +119,8 @@ MultiCameraBaseLauncher::MultiCameraBaseLauncher(ICameraRig& cameraRig, const od
 
   if (!fig.is_valid()) {
     throw std::runtime_error(
-        "Bad calibration. cuVSLAM is supposed to work with at least"
-        " one stereo pair available.");
+        "Bad calibration. cuVSLAM needs at least one configured depth camera or one camera pair with overlapping "
```

Cppcheck (2.21.0)

[style] The function 'add' is never used.

(unusedFunction)